Automattic/wp-calypso
# A4A Referrals: Display success notice when referral submission is pending

Resolves https://github.com/Automattic/jetpack-genesis/issues/334

## Proposed Changes

* When the Tipalti account status is "success", display a success notice at the top of the Referrals page.
* Save when the notice has been displayed, and only show it once.
* Updates the `NoticeBanner` component to allow the "title" prop to be undefined.

## Testing Instructions



* View the `/referrals` screen, and validate the success notice is displayed. Click the close icon to dismiss the notice.
* Refresh the page and confirm the notice is not shown again.

## Pre-merge Checklist




## Screenshots

<img width="1345" alt="Screenshot 2024-04-25 at 11 11 51 AM" src="https://github.com/Automattic/wp-calypso/assets/10933065/b238d2c2-1de3-4293-a6cd-9f849bd1db20">
